src/Ledger/Chain/Properties.agda

```diff
@@ -51,10 +48,9 @@ instance
 
   Computational-CHAIN : Computational _⊢_⇀⦇_,CHAIN⦈_
   Computational-CHAIN .computeProof Γ s b = do
-    _ , neStep ← computeProof _ _ _
+    _ , neStep ← computeProof {STS = _⊢_⇀⦇_,NEWEPOCH⦈_} _ _ _
```

Comment:
This is the only short-coming of this PR, I don't know why though given that this code does not use decidability in any way :S 

@UlfNorell Certainly a long shot, but any idea why this type annotation is necessary?

Reply:
It looks like there's some interaction between the Computational constraint and the Monad constraint from the do-block. If you use explicit `Maybe.>>=` the it solves the Computational constraint. Why it can't figure out that it's the Maybe monad I don't know given that computeProof returns a Maybe.

Reply:
I think the Monad constraint on the second bind gets block (unnecessarily) on unsolved metas in the computeProof call.

Reply:
Do you think it's worth minimizing to a simple example and submitting as an Agda issue?

Reply:
Might not be a bad idea.